### PR TITLE
Fix unmanaged file inspector for redhat

### PR DESCRIPTION
**Pull request by thardeck**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 12cba44b4d8828b98af52a543c75c6ff15a52739 into 4d75214a7046c2f351c267d7b11cec2cfc38366c )

**Comment by tgoettlicher** [comment]
Lgtm although hound complained.

File: lib/tarball.rb [hunk 1]
```diff
@@ -33,6 +33,12 @@ def list
           # This may fail for files with "->" in their name, but there is no way
           # how to avoid this when using "tar".
           path = rest.join(" ").split(" -> ").first
+        when "h"
+          type = :file
+          # This may fail for files with "link to" in their name, but there is no way
+          # how to avoid this when using "tar" unless we use the parameter
+          # --hard-dereference with tar to get rid of hard links
+          path = rest.join(" ").split(" link to ").first
```
Comment (cornelius) [inline lib/tarball.rb]: It would be good to have a unit test for this change.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 12cba44b4d8828b98af52a543c75c6ff15a52739 into 2f9d57695b44ae5634e55ebbeb93084693f4a9fd )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 640de298360e1b777731c04b31e1736f8a49da56 into 2f9d57695b44ae5634e55ebbeb93084693f4a9fd )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 83277943604d77ff0805b3543c135be4bd15553f into 2f9d57695b44ae5634e55ebbeb93084693f4a9fd )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 5b30877841c1acc69bc3daf0330a2d5946a0cdd7 into 2f9d57695b44ae5634e55ebbeb93084693f4a9fd )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged cf5540cc836b59e2da92237ecef29cbbf3e2941d into 2f9d57695b44ae5634e55ebbeb93084693f4a9fd )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 4c79f042c24b80a2b3309de56dc70fcfb8625ba0 into 2f9d57695b44ae5634e55ebbeb93084693f4a9fd )

**Comment by cornelius** [comment]
lgtm

**Pull request merged**